site/server-lifecycle.md

```diff
@@ -132,7 +132,6 @@ The operator runtime does rolling restarts of clustered servers so that service
 
 ### Properties that cause servers to be restarted
 The operator will restart servers when any of the follow properties on the domain resource that affect the server are changed:
-* `annotations`
```

Comment:
Any thoughts on how we should document that servers may have labels or annotations that were previously specified on the domain resource given that the server won't be restarted?

Reply:
I've added a note.